foundationdb/assets/dashboards/foundationdb_transactions_and_queues.json

```diff
@@ -1 +1 @@
ery1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":4,"y":0,"width":4,"height":2}},{"id":6259859862527082,"definition":{"title":"Completed Read operations","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Reads / second","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:foundationdb.workload.operations.reads.hz{$service}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":8,"y":0,"width":4,"height":2}},{"id":3689974919495706,"definition":{"title":"Write Operations Completed","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Writes / second","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:foundationdb.workload.operations.writes.hz{$service}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":0,"y":2,"width":4,"height":2}},{"id":1246848430284440,"definition":{"title":"Proxy memory limit errors","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Errors / second","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:foundationdb.workload.operations.memory_errors.hz{$service}","data_source":"metrics","name":"query1"}],"style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"},"markers":[]},"layout":{"x":4,"y":2,"width":4,"height":2}},{"id":8304484655265424,"definition":{"type":"note","content":"Memory errors are requests rejected by proxies because they are reached their memory limit, which indicates they already have too many requests.","background_color":"white","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":true},"layout":{"x":8,"y":2,"width":4,"height":2}}]},"layout":{"x":0,"y":25,"width":12,"height":5}}],"template_variables":[{"name":"service","default":"*","prefix":"service","available_values":[]}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"6vn-5wq-k3i"}
```

Comment:
Small nits: 

> {"id":4772159496539460,"definition":{"type":"note","content":"The **numbers** of [transactions](https://apple.github.io/foundationdb/developer-guide.html#transaction-basics) started and committed give an overall idea of the throughput of the database cluster.

Can we change that to "The **number** of [transactions]". (Bold added for emphasis)

> Such a problem may indicate that the storage server cannot hold the write throughput; if it afflicts only a particular storage server, perhaps the writes are heavily focused on a certain key.

Can we change the semicolon to a period? "...hold the write throughput. If it ..."

Reply:
Sure, good catches!

Reply:
Sorry, can we also capitalize the `I` in `if`?

Reply:
My bad 🤦‍♀️